### PR TITLE
lesson 14 done

### PR DESCRIPTION
**Pull request by ilyabaranowski**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,12 +5,16 @@
...
     <title>Document</title>
+    <link rel="stylesheet" href="./main.css">
 </head>
 
 <body>
-    <script src="./main.js"></script>
+    <form class="formCSS" id="formJS">
+        <input type="file" name="file">
+        <input type="submit" name="send">
+        <script src="./main.js"></script>
```
Comment (nikitik-nikita) [inline index.html]: Script in wrong place

File: main.js [hunk 2]
```diff
@@ -1,56 +1,23 @@
...
 
-const dataLoves = async () => {
-  const dataMain = await hotelsData('https://fe-student-api.herokuapp.com/api/hotels/popular');
+  const fetchInfo = {
+    method: 'POST',
+    body: new FormData(formEl),
+  };
 
-  const wrapper = document.querySelector('body');
+  const resp = await fetch('https://fe-student-api.herokuapp.com/api/file.', fetchInfo)
```
Comment (nikitik-nikita) [inline main.js]: Wrong url. It should be `'https://fe-student-api.herokuapp.com/api/file'`

File: index.html (hunk 1 above)
Comment (ilyabaranowski) [reply index.html]: fixed

File: main.js (hunk 2 above)
Comment (ilyabaranowski) [reply main.js]: fixed

**Pull request merged**